xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -58,19 +58,24 @@ class CourseContentTableViewController: UITableViewController {
     }
 
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        let courseItem = sender as! CourseItem
```

Comment:
Don't force-cast here. Either do it in the cases or do an optional cast (`as? CourseItem`).